sites/partners/__tests__/pages/settings/index.test.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react"
+import { setupServer } from "msw/lib/node"
+import { fireEvent, within } from "@testing-library/react"
+import Settings from "../../../pages/settings"
+import { rest } from "msw"
+import { listing, multiselectQuestionPreference } from "../../testHelpers"
+import { render } from "../../testUtils"
+
+const server = setupServer()
+
+beforeAll(() => {
+  server.listen()
+})
+
+afterEach(() => server.resetHandlers())
+
+afterAll(() => server.close())
+
+describe("settings", () => {
+  it("should render `none` when no preferences exist", async () => {
+    server.use(
+      rest.get("http://localhost:3100/multiselectQuestions", (_req, res, ctx) => {
+        return res(ctx.json([]))
+      })
+    )
+
+    const { getByText, findByText } = render(<Settings />)
+
+    expect(getByText("Settings")).toBeInTheDocument()
+    expect(getByText("Preferences")).toBeInTheDocument()
+
+    await findByText("None")
```

Comment:
Is this just to ensure it loads?

Reply:
Yeah, it verifies that when the empty array is returned the component outputs "None". Since we have to wait for the API call to return it's safer to do the `await findBy....`